functions/src/scripts/convert-tag-to-group.ts

```diff
@@ -0,0 +1,66 @@
+// Takes a tag and makes a new group with all the contracts in it.
+
+import * as admin from 'firebase-admin'
+import { initAdmin } from './script-init'
+import { isProd, log } from '../utils'
+import { getSlug } from '../create-group'
+import { Group } from '../../../common/group'
+
+const getTaggedContractIds = async (tag: string) => {
+  const firestore = admin.firestore()
+  const results = await firestore
+    .collection('contracts')
+    .where('lowercaseTags', 'array-contains', tag.toLowerCase())
+    .get()
+  return results.docs.map((d) => d.id)
+}
+
+const createGroup = async (
+  name: string,
+  about: string,
+  contractIds: string[]
+) => {
+  const firestore = admin.firestore()
+  const creatorId = isProd()
+    ? 'IPTOzEqrpkWmEzh6hwvAyY9PqFb2'
+    : '94YYTk1AFWfbWMpfYcvnnwI1veP2'
```

Comment:
This could probably be a constant for the environment. I created a function `isManifoldId` in `constants.ts` which could also be updated based on the environment. Probably something for a followup PR.

Reply:
Yes, I did all that but I reverted it because I got too confused and didn't feel like dealing with it right now.